quint/src/runtime/impl/runtimeValue.ts

```diff
@@ -154,7 +154,9 @@ export const rv = {
    */
   mkMap: (elems: Iterable<[RuntimeValue, RuntimeValue]>): RuntimeValue => {
     // convert the keys to the normal form, as they are hashed
-    const arr: [RuntimeValue, RuntimeValue][] = Array.from(elems).map(([k, v]) => [k.normalForm(), v])
+    const arr: [RuntimeValue, RuntimeValue][] = Array.from(elems).map(([k, v]) => {
+      return [k.normalForm(), v] // Ensure normalForm works with tuples
```

Comment:
I think this was a leftover?
```suggestion
      return [k.normalForm(), v]
```